apps/wpcom-block-editor/src/wpcom/features/deprecate-coblocks-buttons.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+/**
+ * External dependencies
+ */
+import { getBlockType, registerBlockType, unregisterBlockType } from '@wordpress/blocks';
+import domReady from '@wordpress/dom-ready';
```

Comment:
I looked at a related Jetpack PR and noticed the `domReady` script dependency was added. Is that related to this change? Are these dependencies externals? If so, do we have a way of ensuring that the dependencies are properly enqueued when the script is served from the widgets URL?

Reply:
Good question. Jetpack always loads the latest bundle from widgets URL, no matter what version of Jetpack we're running.

Reply:
> I looked at a related Jetpack PR and noticed the `domReady` script dependency was added. Is that related to this change? 

Do you mean https://github.com/Automattic/jetpack/pull/14310? If so, yeah, it was related to a previous commit of this PR. We changed the approach and that PR was no longer needed, since this `domReady` is used in the `wpcom` bundled script which [Jetpack already enqueues with `wp-dom-ready` as dependency](https://github.com/Automattic/jetpack/blob/572dd6488baf30ef15853ca0cbb30a8f98294f16/modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php#L306).

> Are these dependencies externals? 

Yup, they are [externalized when generating the build with Webpack](https://github.com/Automattic/wp-calypso/blob/b08d9f6422751c7f4a7e78aba0e5a9a75aa2c9b7/apps/wpcom-block-editor/webpack.config.js#L63).

> If so, do we have a way of ensuring that the dependencies are properly enqueued when the script is served from the widgets URL?

We manually maintain them in [Jetpack](https://github.com/Automattic/jetpack/blob/572dd6488baf30ef15853ca0cbb30a8f98294f16/modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php#L258-L344) and WP.com (handled in the `gutenberg_wpcom_enqueue_scripts` function of the `wp-content/plugins/gutenberg-wpcom/index.php` file).